src/components/CounterBubble/CounterBubble.vue

```diff
@@ -62,10 +62,22 @@ export default {
 				return ['highlighted', 'outlined', ''].indexOf(value) !== -1
 			},
 		},
+
+		/**
+		 * DEPRECATED: use the prop "type" instead.
+		 * Toggles the highlighted state of the counter.
+		 */
+		counterHighlighted: {
```

Comment:
The prop was just called `highlighted` in the `CounterBubble` component before https://github.com/nextcloud/nextcloud-vue/pull/2208 (if I am not wrong)
```suggestion
		highlighted: {
```